templates/email/default/waste/bulky-reminder.html

```diff
@@ -0,0 +1,72 @@
+[%
+
+USE pounds = format('%.2f');
+PROCESS '_bulky_data.html';
+
+email_columns = 2;
+
+PROCESS '_email_settings.html';
+INCLUDE '_email_top.html';
+
+%]
+
+<th style="[% td_style %][% primary_column_style %]" id="primary_column">
+  [% start_padded_box | safe %]
+
+  <p style="[% p_style %]">Dear [% report.name %],</p>
+
+  <p style="[% p_style %]">[% email_summary %]</p>
+
+  <p style="[% p_style %]">
+    [% IF days == 3 %]
+    This is a reminder that your collection is in
+    <strong>3 days</strong>.
+    [% ELSIF days == 1 %]
+    This is a reminder that your collection is
+    <strong>tomorrow</strong>.
+    [% END %]
+  </p>
+
+    [% INCLUDE '_council_reference.html' problem=report %]
+
+  <p style="[% p_style %]">
+    Collection date: [% collection_date %]
+  </p>
+
+  <p style="[% p_style %]">
+    Items to be collected:<br>
+    [% item_list.join('<br>') %]
```

Comment:
Not quite sure what's gone on here, but might want a `| safe` at the very least! 
<img width="290" alt="image" src="https://user-images.githubusercontent.com/4776/204297424-7f03f4ce-b07e-48ae-8f57-04df43091eb1.png">

Reply:
Ah, it looks like you might have up to ITEM_20? And I'm not filtering out blanks.

Reply:
Yep, that's exactly what was happening 👍